connector/mqtt/vernemq/broker/bin/vmq_dojot/scripts_tls/checkCertificateChain.sh

```diff
@@ -4,11 +4,11 @@ BASE_DIR=${BASE_DIR:-"/vernemq"}
 
 . "${BASE_DIR}"/scripts_tls/_initVariables.sh
 
-if openssl verify -CAfile "${certDir}"/"${certCaFile}" "${certDir}"/"${certCertFile}" 
-then 
+if openssl verify -CAfile "${certDir}"/"${certCaFile}" "${certDir}"/"${certCertFile}"
```

Comment:
Can they be modified to: 
"${certDir}"/"${certCaFile}"  -> "${certDir}/${certCaFile}" 
"${certDir}"/"${certCertFile}" -> "${certDir}/${certCertFile}"